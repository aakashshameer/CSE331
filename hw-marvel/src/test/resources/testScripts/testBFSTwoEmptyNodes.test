# Test BFS on twoEmptyNodes graph

# build the graph
LoadGraph twoEmptyNodes twoEmptyNodes.csv

# test BFS order

# find the shortest path A to B
FindPath twoEmptyNodes A B

# find the shortest path B to A
FindPath twoEmptyNodes B A