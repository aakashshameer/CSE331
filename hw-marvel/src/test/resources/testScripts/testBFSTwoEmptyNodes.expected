# Test BFS on twoEmptyNodes graph

# build the graph
loaded graph twoEmptyNodes

# test BFS order
# find the shortest path A to B
path from A to B:
no path found

# find the shortest path B to A
path from B to A:
no path found